framework/PageHeader.tsx

```diff
@@ -107,6 +109,31 @@ export function PageHeader(props: PageHeaderProps) {
   const isXl = useBreakpoint('xl');
   const isMdOrLarger = useBreakpoint('md');
   const [translations] = useFrameworkTranslations();
+  const { pageBreadcrumbs, baselineBreadcrumbs, setBaselineBreadcrumbs, setPageBreadcrumbs } =
+    usePageBreadcrumbs();
+
+  useEffect(() => {
+    if (breadcrumbs?.length) {
+      setBaselineBreadcrumbs(breadcrumbs ?? []);
+      setPageBreadcrumbs([...baselineBreadcrumbs]);
```

Comment:
not sure I understand this bit...

if breadcrumbs were passed from props, this sets baseline breadcrumbs to those, and then uses the previous value of baseline breadcrumbs to set page breadcrumbs?

Reply:
I had added this for the cases where PageHeader is used independently from PageRoutedTabs. So that pageBreadcrumbs default to the baseline breadcrumbs if there are no PageRoutedTabs to go along with it.